cmd/uplink/cmd/cp.go

```diff
@@ -30,13 +30,22 @@ var (
 	})
 )
 
+func cleanAbsPath(path string) string {
+	prefix := strings.HasSuffix(path, "/")
```

Comment:
I am quite confused what this method does. Why naming a variable `prefix` if the check is related to a suffix.